<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfazSIR</class>
 <widget class="QWidget" name="InterfazSIR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="opcionesLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="1" column="0">
     <layout class="QGridLayout" name="datosLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="leS"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="leR"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lDInf">
        <property name="text">
         <string>DInf</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="leDInf"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lS">
        <property name="text">
         <string>S</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lC">
        <property name="text">
         <string>C</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="leTC"/>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="leN"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="lDInc">
        <property name="text">
         <string>DInc</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="lR">
        <property name="text">
         <string>R</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="leDInc"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="leC"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="lTI">
        <property name="text">
         <string>TC</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lN">
        <property name="text">
         <string>N</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="leM"/>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="lM">
        <property name="text">
         <string>M</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="rbDatos">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>Ingresar Datos</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="rbHistorico">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>Calcular datos desde histórico</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="teDatos">
      <property name="plainText">
       <string>Se usarán los siguientes datos:

* Dur. infección: 7 días
* Dur. incubación:  6 días
* Tasa de contagios:   2.68</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbGraficar">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Graficar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbPeriodo">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>700</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="lPeriodo">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad de días a calcular</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabGraficos">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>139</y>
     <width>781</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="Grafico" name="tabCRM">
    <attribute name="title">
     <string>Modelo CRM</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="Grafico" name="tabSIRM">
    <attribute name="title">
     <string>Modelo SIRM</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbSalir">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Grafico</class>
   <extends>QWidget</extends>
   <header location="global">grafico.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rbDatos</sender>
   <signal>toggled(bool)</signal>
   <receiver>leN</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbDatos</sender>
   <signal>toggled(bool)</signal>
   <receiver>leS</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbDatos</sender>
   <signal>toggled(bool)</signal>
   <receiver>leC</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbDatos</sender>
   <signal>toggled(bool)</signal>
   <receiver>leR</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbDatos</sender>
   <signal>toggled(bool)</signal>
   <receiver>leM</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbDatos</sender>
   <signal>toggled(bool)</signal>
   <receiver>leDInf</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbDatos</sender>
   <signal>toggled(bool)</signal>
   <receiver>leDInc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbDatos</sender>
   <signal>toggled(bool)</signal>
   <receiver>leTC</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbGraficar</sender>
   <signal>pressed()</signal>
   <receiver>InterfazSIR</receiver>
   <slot>iniciarGraficos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSalir</sender>
   <signal>pressed()</signal>
   <receiver>InterfazSIR</receiver>
   <slot>salir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>738</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>iniciarGraficos()</slot>
  <slot>salir()</slot>
 </slots>
</ui>
